packages/wasm/src/main.ts

```diff
@@ -1,23 +1,27 @@
 import "./style.css"
 // import '../dist/index.d.ts'
 // import { VGG, EventType } from "../dist/vgg-wasm.js"
-import { VGG, EventType } from "../lib/main"
-import { State } from "../lib/constants"
+import { VGG } from "../lib/main"
 // import { Generated_Nodes_Type } from "./main.d"
 
 document.querySelector<HTMLDivElement>("#app")!.innerHTML = `
   <div>
     <h1>VGG Wasm</h1>
     <canvas id="canvas"></canvas>
+    <div id="frames"></div>
+    <button id="snapshot">Snapshot</button>
+    <div id="prev">Prev</div>
+    <div id="next">Next</div>
   </div>
 `
 
 const vgg = new VGG({
   // src: "https://s3.vgg.cool/test/vgg.daruma",
   // src: "https://verygoodgraphics.com/d/clqywc8tu002bo9mn8bctgekn",
-  src: "https://raw.githubusercontent.com/verygoodgraphics/resource/main/feature/geometry/geometry__transform__rotation.daruma",
+  // src: "https://raw.githubusercontent.com/verygoodgraphics/resource/main/feature/geometry/geometry__transform__rotation.daruma",
+  src: "http://localhost:3030/d/clv1tt4j60008i80yxnqjhysc",
```

Comment:
Better if delete this localhost links because it's not available on other's computer.